src/newSeed/baseStage.ts

```diff
@@ -31,6 +31,7 @@ export abstract class BaseStage {
   protected maxStage: number;
   protected stageStates: Array<IStageState>;
   protected wizardState: IWizardState;
+  protected nextButtonClicked: boolean;
```

Comment:
Here I am keeping track if the next button was clicked, this way I can tell that the validate input function has been called